docs/source/userguide.rst

```diff
@@ -32,6 +32,59 @@ for a source named "my_source",
 
     iact-estimator --config config.yml --source-name my_source
 
+Using custom performance data
+=============================
+
+If you want to use your own performance data,
+either for testing or because it is still not published,
+you can load it from an ECSV with the format shown below.
+
+Producing such a table is very easy with astropy
+starting from the array quantities containing the data,
+for example,
+
+.. code-block:: python
+
+    from astropy.table import QTable
+    import astropy.units as u
+    import numpy as np
+
+    table = QTable([min_energy, max_energy, gamma_rate, bkg_rate],
+                    names=("min_energy", "max_energy", "gamma_rate", "background_rate"),
+                    meta={"name":"some_descriptive_title"})
+    table.write("my_performance.ecsv")
+
+this will result in a data file similar to this,
+
+.. code-block::
+
+    # %ECSV 1.0
+    # ---
+    # datatype:
+    # - {name: min_energy, unit: GeV, datatype: float64}
+    # - {name: max_energy, unit: GeV, datatype: float64}
+    # - {name: gamma_rate, datatype: float64}
+    # - {name: background_rate, datatype: float64}
```

Comment:
I think it would be nice to add unit for gamma_rate and background_rate as examples.

Reply:
should be better now, I basically copied the header from the MAGIC low energy data